llvm/unittests/DebugInfo/PDB/Inputs/PublicSymbols.cpp

```diff
@@ -0,0 +1,46 @@
+// clang-format off
```

Comment:
Is the goal of this file to generate the `PublicSymbols.pdb` below? We usually don't keep artifacts like this around, if they are not directly used by a test. As for the binary PDB below, I think you'd have to actually build a `PublicsStream` programatically in the unit test, without ever going through a PDB file. Can you try doing that? I am not sure all APIs for doing that are public, but worth a try. That could also allow for covering the "corrupted" stream case.